### DISCUSSION TITLE
About Incident Recovery

### DISCUSSION THREAD

**Discussion by 1998729** [jertel/elastalert2#1019] [Q&A] [answered]
Does elastalert2 also not support event recovery? If I use alertmanager, alertmanager will do this for me. He should regularly check whether this event will continue to occur, and if not, it has been restored.

**Comment by 1998729** [comment]
<img width="831" alt="image" src="https://user-images.githubusercontent.com/88186802/202608006-cea69864-4cd2-468d-96ed-5e7b3a5d1eb1.png">

**Comment by 1998729** [comment]
@nsano-rururu

**Comment by nsano-rururu** [comment, marked as answer]
I've never used elastalert2 and am not familiar with alertmanager either. I certainly added the alert manager alert notification, but I just made a pull request to elastalert2 with some changes and corrections based on the original yelp/elastalert pull request.
At that time, I only moved it lightly to check the operation.
